fulp_modules/features/species/beefmen/beefman_clothing.dm

```diff
@@ -41,113 +41,113 @@
 
 /obj/item/clothing/under/bodysash/security/detective
 	name = "detective's sash"
-	desc = "A sash for someone that mean business."
+	desc = "A sash for someone who means business."
 	icon_state = "detective"
 
 /obj/item/clothing/under/bodysash/security/deputy
 	name = "deputy's sash"
-	desc = "An awe-inspiring \"tactical\" sash; because safety never takes a holiday."
+	desc = "An awe-inspiring \"tactical\" sash— because safety never takes a holiday."
 	icon_state = "deputy"
 
 //Medical
 /obj/item/clothing/under/bodysash/medical
 	name = "medical's sash"
-	desc = "A doctor's sash, It's made of a special fiber that provides minor protection against biohazards.."
```

Comment:
why are you dropping the special fiber text, goes for them all

Reply:
As a detail it seems neither necessary nor specific enough to actually add anything to the text. People who know that the clothing is resistant to something will automatically assume it's because of the clothing's material, but they might not know how exactly that material makes it resistant.